src/c++/perf_analyzer/main.cc

```diff
@@ -589,19 +604,78 @@ Usage(char** argv, const std::string& msg = std::string())
             << std::endl;
   std::cerr << std::endl;
   std::cerr << "III. SERVER DETAILS: " << std::endl;
-  std::cerr << std::setw(9) << std::left << " -u: "
+  std::cerr << std::setw(38) << std::left << " -u: "
```

Comment:
38 is everywhere. It should be a const

Reply:
These magic numbers are currently used everywhere in this file. I would prefer to save this kind of refactor work for another time/PR. I've created a Jira issue for this 👍

Reply:
Seems like a boyscout thing that should have been done when you touched the existing numbers, but since this is needed by EOW we can move to a follow-up story